packages/itwinui-react/src/core/utils/components/LinkBox.tsx

```diff
@@ -0,0 +1,44 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import cx from 'classnames';
+import {
+  PolymorphicComponentProps,
+  PolymorphicForwardRefComponent,
+} from '../props';
+
+export type PolymorphicActionProps = PolymorphicComponentProps<
+  'a',
+  // eslint-disable-next-line @typescript-eslint/ban-types
+  {}
+>;
+
+/**
+ * Polymorphic action component.
+ * It is rendered as `a` by default.
+ */
+export const LinkOverlay = React.forwardRef((props, ref) => {
```

Comment:
will need tests for this.

also nit: i think filename should be `LinkOverlay` because it is the "meaty" part of the component. `LinkBox` is a simple wrapper.